frontend/src/app/page-not-found/page-not-found.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="not-found">
+  <h1>404 Hakemaasi sivua ei löydy. Tarkista sivun osoite.</h1>
```

Comment:
**thought:** doesn't matter too much but 404 means nothing to users in general

Reply:
Let's delete this component and instead redirect users to the home page.